frontend/src/components/JiraImport.svelte

```diff
@@ -0,0 +1,70 @@
+<script>
+    import HollowButton from '../components/HollowButton.svelte'
+
+    export let notifications
+    export let handlePlanAdd = () => {}
+
+    let files
+
+    function showDialog () {
```

Comment:
I'd rather use css hack to overlay the file input on the button and turn its opacity to 0
https://www.quirksmode.org/dom/inputfile.html

Reply:
I checked this solution. And I cannot be convinced of his work. It doesn't work in google chrome 85.0.4183.121 on mac

Reply:
Alright I will look at optimizing it post-merge